From 1815337f7dbb530ae3e3bc1175d8da214cf76eb5 Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
Date: Tue, 9 Mar 2021 14:19:12 +0000
Subject: [PATCH] spec: bump version to 2.0

The shared library is now liburing.so.2 but the pkgconfig and rpm files
still say 0.7. Existing binaries link against liburing.so.1 and will not
automatically pick up the new liburing.so.2 shared library.

Update the version number in liburing.spec so

Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
Link: https://lore.kernel.org/r/20210309141913.262131-2-stefanha@redhat.com
Signed-off-by: Jens Axboe <axboe@kernel.dk>
---
 liburing.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/liburing.spec b/liburing.spec
index fa4d9704..8607074c 100644
--- a/liburing.spec
+++ b/liburing.spec
@@ -1,5 +1,5 @@
 Name: liburing
-Version: 0.7
+Version: 2.0
 Release: 1%{?dist}
 Summary: Linux-native io_uring I/O access library
 License: (GPLv2 with exceptions and LGPLv2+) or MIT
